README.rst

```diff
@@ -188,6 +193,41 @@ You need to put `rest_tornado` in the configuration file.
 
 Please be aware that depending on the salt-api implementation you use, saltpad may requires a different version of salt on the master side, please refer to the table at the beginning of the Readme.
 
+Rest_cherrypy single-app page deployment
+----------------------------------------
+
+The rest_cherrypy flavour allow the deployment of single-app pages directly by the salt-api. The salt-api needs to be configured this way:
+
+.. code-block:: ini
+
+    rest_cherrypy:
+      port: 8000
+      static: /code/static
+      static_path: /static
+      app: /code/index.html
+      app_path: /saltpad
+
+This configuration is valid for a deployment of saltpad release archive on /code. The `index.html` file needs to be accessible on path `/code/index.html` and readable by salt-api. The `app_path` setting key must be different than `/` and must_correspond to the `PATH_PREFIX` saltpad configuration key. Here is the corresponding saltpad configuration file:
```

Comment:
This configuration is valid for a deployment of saltpad release archive on /code. The `index.html` file needs to be accessible on path `/code/index.html` and readable by salt-api. The `app_path` setting key must be different than `/` and must correspond to the saltpad configuration key `PATH_PREFIX`. Here is the corresponding saltpad configuration file: